diff -ub compile.js new-compile.js
--- compile.js	2013-11-20 17:11:09.000000000 +0100
+++ new-compile.js	2013-11-27 15:29:34.336013246 +0100
@@ -1,7 +1,7 @@
 var po = require('node-po');

 var template = function (module, body) {
-    return "angular.module(\"" + module + "\").run(['gettextCatalog', function (gettextCatalog) {\n" + body + "\n}]);";
+    return "/* jshint -W100 */\nangular.module(\"" + module + "\").run(['gettextCatalog', function (gettextCatalog) {\n" + body + "\n}]);\n/* jshint +W100 */";
 };

 var langTemplate = function (language, strings) {
diff -ub extract.js new-extract.js
--- extract.js	2013-11-21 12:02:18.000000000 +0100
+++ new-extract.js	2013-11-27 15:31:41.240630362 +0100
@@ -119,7 +119,22 @@
             catalog.items.sort(function (a, b) {
                 return a.msgid.localeCompare(b.msgid);
             });
-
+            function DateString(d){
+              function pad(n){return n<10 ? '0'+n : n}
+              return d.getUTCFullYear()+'-'
+                  + pad(d.getUTCMonth()+1)+'-'
+                  + pad(d.getUTCDate())+' '
+                  + pad(d.getUTCHours())+':'
+                  + pad(d.getUTCMinutes())+':'
+                  + pad(d.getUTCSeconds())+''
+                  + (d.getTimezoneOffset() < 0 ? '-' : '+')
+                  + pad(Math.floor(d.getTimezoneOffset() / 60) * (d.getTimezoneOffset() < 0 ? -1 : 1))
+                  + pad(d.getTimezoneOffset() % 60)
+            }
+            catalog.headers['POT-Creation-Date'] = DateString(new Date());
+            catalog.headers['Content-Type'] = 'text/plain; charset=UTF-8';
+            catalog.headers['Plural-Forms'] = 'nplurals=INTEGER; plural=EXPRESSION;';
+            catalog.headers['Content-Transfer-Encoding'] = '8bit';
             if (!failed) {
                 grunt.file.write(file.dest, catalog.toString());
             }
