diff -ub compile.js new-compile.js
--- compile.js	2013-10-09 16:17:40.257875241 +0200
+++ new-compile.js	2013-11-04 10:50:04.039919981 +0100
@@ -3,7 +3,7 @@
 po = require('node-po');

 template = function(module, body) {
-  return "angular.module(\"" + module + "\").run(['gettextCatalog', function (gettextCatalog) {\n" + body + "\n}]);";
+  return "/* jshint -W100 */\nangular.module(\"" + module + "\").run(['gettextCatalog', function (gettextCatalog) {\n" + body + "\n}]);\n/* jshint +W100 */";
 };

 langTemplate = function(language, strings) {
diff -ub extract.js new-extract.js
--- extract.js	2013-11-04 10:48:16.571666811 +0100
+++ new-extract.js	2013-11-04 10:49:47.770184495 +0100
@@ -106,6 +106,22 @@
         string = strings[key];
         catalog.items.push(string);
       }
+      function DateString(d){
+        function pad(n){return n<10 ? '0'+n : n}
+        return d.getUTCFullYear()+'-'
+            + pad(d.getUTCMonth()+1)+'-'
+            + pad(d.getUTCDate())+' '
+            + pad(d.getUTCHours())+':'
+            + pad(d.getUTCMinutes())+':'
+            + pad(d.getUTCSeconds())+''
+            + (d.getTimezoneOffset() < 0 ? '-' : '+')
+            + pad(Math.floor(d.getTimezoneOffset() / 60) * (d.getTimezoneOffset() < 0 ? -1 : 1))
+            + pad(d.getTimezoneOffset() % 60)
+      }
+      catalog.headers['POT-Creation-Date'] = DateString(new Date());
+      catalog.headers['Content-Type'] = 'text/plain; charset=UTF-8';
+      catalog.headers['Plural-Forms'] = 'nplurals=INTEGER; plural=EXPRESSION;';
+      catalog.headers['Content-Transfer-Encoding'] = '8bit';
       if (!failed) {
         return grunt.file.write(file.dest, catalog.toString());
       }
