--- extract.js      2013-10-04 14:54:23.097745289 +0200
+++ new-extract.js  2013-10-04 14:55:42.109821844 +0200
@@ -119,6 +119,22 @@
         string = strings[key];
         catalog.items.push(string);
       }
+      function DateString(d){
+        function pad(n){return n<10 ? '0'+n : n}
+        return d.getUTCFullYear()+'-'
+            + pad(d.getUTCMonth()+1)+'-'
+            + pad(d.getUTCDate())+' '
+            + pad(d.getUTCHours())+':'
+            + pad(d.getUTCMinutes())+':'
+            + pad(d.getUTCSeconds())+''
+            + (d.getTimezoneOffset() < 0 ? '-' : '+')
+            + pad(Math.floor(d.getTimezoneOffset() / 60) * (d.getTimezoneOffset() < 0 ? -1 : 1))
+            + pad(d.getTimezoneOffset() % 60)
+      }
+      catalog.headers['POT-Creation-Date'] = DateString(new Date());
+      catalog.headers['Content-Type'] = 'text/plain; charset=UTF-8';
+      catalog.headers['Plural-Forms'] = 'nplurals=INTEGER; plural=EXPRESSION;';
+      catalog.headers['Content-Transfer-Encoding'] = '8bit';
       if (!failed) {
         return grunt.file.write(file.dest, catalog.toString());
       }
